packages/strapi-provider-upload-cloudinary/lib/index.js

```diff
@@ -16,8 +16,14 @@ module.exports = {
     return {
       upload(file, customConfig = {}) {
         return new Promise((resolve, reject) => {
+          const config = {
+            resource_type: 'auto',
+            public_id: file.hash,
+            filename: `${file.hash}${file.ext}`,
```

Comment:
what happens in case there is no extension on the file ? looks like we are gonna get sth like 'fileundefined'

Reply:
That's true, I'll change that